scene/resources/environment.cpp

```diff
@@ -31,6 +31,9 @@
 #include "environment.h"
 
 #include "core/config/project_settings.h"
+#include "core/core_string_names.h"
+#include "core/error/error_macros.h"
```

Comment:
Are the additional includes needed?

Reply:
These are used for set_adjustment_color_correction's gradient signals.